index.bs

```diff
@@ -482,51 +482,34 @@ Issue: TODO: Handle subresource requests and iframes for client-side redirects.
 </div>
 
 Note: We currently don't treat cookie reads as stateful, but this would be a
-reasonable future change. We could run [=process a network cookie access for bounce tracking mitigations=]
+reasonable future change. We could run [=process a navigation storage access for bounce tracking mitigations=]
 in the <a spec="fetch">HTTP-network-or-cache fetch</a> algorithm after step 8.21.1.2,
 "... [=append=] (`Cookie`, cookies) to httpRequest’s [=header list=]. ..."
 
 <h5 id="bounce-tracking-mitigations-service-worker-activation-monkey-patch">Service Worker Activation Monkey Patch</h5>
 
 Each [=top-level traversable=] maintains a record of which sites have activated service workers in the current [=extended navigation=].
 
-Insert the following steps in the [activate]((https://w3c.github.io/ServiceWorker/#activate) algorithm before step 11,
-"Run the [Update Worker State](https://w3c.github.io/ServiceWorker/#update-worker-state) algorithm...":
+Insert the following steps in the [Handle Fetch](https://w3c.github.io/ServiceWorker/#on-fetch-request-algorithm) algorithm after step 23,
+"If the result of running the [Run Service Worker](https://w3c.github.io/ServiceWorker/#run-service-worker) algorithm...":
 
-1. Run [=process a service worker activation for bounce tracking mitigations=] given <var ignore>registration</var> and <var ignore>matchedClients</var>.
-
-<div algorithm>
-
-To <dfn>process a service worker activation for bounce tracking mitigations</dfn>
-given a [=service worker registration=] |registration| and a list of [=service worker clients=] |matchedClients|,
-perform the following steps:
-
-1. Let |host| be |registration|'s [=service worker registration/scope url=]'s [=host=].
-1. [=list/For each=] |client| in |matchedClients|:
-    1. Let |browsingContext| be |client|'s [=environment/target browsing context=].
-    1. If |browsingContext| is null, then [=iteration/continue=].
-    1. Let |topLevelTraversable| be |browsingContext|'s [=browsing context/top-level traversable=].
-    1. If |topLevelTraversable|'s [=top-level traversable/bounce tracking record=] is null, then [=iteration/continue=].
-    1. [=set/Append=] |host| to |topLevelTraversable|'s [=top-level traversable/bounce tracking record=]'s
-        [=bounce tracking record/storage access set=].
-
-</div>
+1. Run [=process a navigation storage access for bounce tracking mitigations=] given <var ignore>request</var>.
```

Comment:
For some reason github is not showing me the steps for this algorithm anywhere, but I can see them in the preview.

The preview says that we are looking at the request's client.  I think for this case we actually want reserved client instead:

https://fetch.spec.whatwg.org/#concept-request-reserved-client

The reserved client is the target container for the new document.  The client is the container of the document that initiated the navigation.  Sometimes they are the same (click a link in a tab to navigate the tab), but sometimes they are different (click a link that opens a new tab).

Reply:
"reserved client" is only used for document and iframe requests.  Its confusing.

So maybe the cookie access thing needs to check one or the other depending on destination.